examples/examples2d.jl

```diff
@@ -382,3 +382,64 @@
         render(A)
     end
 end
+
+@block AnshulSinghvi ["colors"] begin
+
+    @cell "Line with varying colors" [lines, colors, colorlegend] begin
+
+        using AbstractPlotting, # duh :)
+              ColorSchemes      # colormaps galore
+
+        AbstractPlotting.__init__();
```

Comment:
Please remove the init's... 
And also remove using AbstractPlotting, the examples are expected to be executed after makie is loaded ;)

Reply:
Will do! I added the init's to make it easier to run on precompiled Makie, but I guess most people don't run that.

Reply:
Woops, will remove `using AbstractPlotting` in a bit when I get back to my laptop